content/build-tool.md

```diff
@@ -273,6 +273,26 @@ Then we can install any npm CSS processing packages that we'd like to use and re
 
 After doing the above, you'll need to ensure you `npm install` and restart the `meteor` process running your app to make sure the PostCSS system has had a chance to set itself up.
 
+<h2 id="hot-module-replacement">Hot Module Replacement</h2>
+
+In Meteor apps, javascript, typescript, css files that are dynamically imported, and many other types of files are converted into javascript modules during the build process. Instead of reloading the client after a rebuild, Meteor is able to update the javascript modules within the running application that were modified. This reduces the feedback cycle while developing by allowing you to view and test your changes quicker.
+
+Hot module replacement (HMR) can be enabled by adding the [hot-module-replacement](https://docs.meteor.com/packages/hot-module-replacement.html) package to your app:
+
+```
+meteor add hot-module-replacement
+```
+
+Many types of javascript modules can not be updated with HMR, so HMR has to be configured to know which modules can be replace and how to replace them. Most apps never need to do this manually. Instead, you can use integrations that configure HMR for you:
```

Comment:
should be 'can be replaced'

Reply:
I meant `can not be replaced`. Do you think it makes more sense to instead be something like `Only some types of modules can be updated with HMR, so HMR has to be configured ...`

Reply:
I meant just that the replace is missing a 'd' in the sentence:
`HMR has to be configured to know which modules can be replace and...`

Reply:
Sorry, I misunderstood. It is fixed.